website/docs/deploy/configuring-unleash-v3.md

```diff
@@ -40,10 +40,11 @@ unleash.start(unleashOptions);
   - _database_ - the database name to be used (`DATABASE_NAME`)
   - _ssl_ - an object describing ssl options, see https://node-postgres.com/features/ssl (`DATABASE_SSL`, as a stringified json object)
   - _version_ - the postgres database version. Used to connect a non-standard database. Defaults to `undefined`, which let the underlying adapter to detect the version automatically. (`DATABASE_VERSION`)
-  - _pool_ - an object describing pool options, see https://knexjs.org/#Installation-pooling. We support the following three fields:
+  - _pool_ - an object describing pool options, see http://knexjs.org/guide/#pool. We support the following four fields:
```

Comment:
This link seems to be dead. The updated link appears to contain what I expect it to contain but more eyes from someone who was familiar with what the old link pointed to would be great!

Reply:
(the diff here is stale, I've updated the links to point to their HTTPS versions)